agents/human_agent.py

```diff
@@ -8,8 +9,31 @@ class HumanAgent(Agent):
     the user choose the action.
     TODO - implement everything!
     """
+
     def __init__(self, name):
         super(HumanAgent, self).__init__(name)
 
     def choose_action(self, env):
-        pass
\ No newline at end of file
+        # create list of available moves
+        available_moves = env.available_moves(self)
+
+        # check if the list are not empty
+        if available_moves:
+
+            # action of the player
+            while True:
+                print("Please choose your action from the list below and press ENTER:")
+                for act in available_moves:
+                    print(act)
+                move = input()
+
+                # check input
+                if move == 'q':
+                    sys.exit(0)
+                try:
+                    assert int(move) in available_moves
+                    return int(move)
+                except:
+                    print("Wrong input, try again or quit (q)")
+
+        print("No moves available for player {}".format(self))
```

Comment:
And here, you should return `None`.

Reply:
func without return statement should return automatically "None". Not? Or do you want me to write it manually?

Reply:
Return `None` Explicitly please.